class InternalError(Exception):
    pass

class PlaceNotFound(Exception):
    pass

class PlaceAlreadyExists(Exception):
    pass

class PlaceUnknown(Exception):
    pass